### PR TITLE
Container state control

### PR DESCRIPTION
**Pull request by ngmiller**
These changes incorporate some basic container state control into the system. From an instance page, one can now click on a container ID and get redirected to a container page. The container page (shown below) allows for some basic control (starting, stopping, pausing, un-pausing) and relays the detailed container response model.

The `instanceModel` now stores the basic summary of a container (returned from `/containers/json`) as well as the detailed model (from `/containers/{id}/json`), which is fetched upon request.

screen shot 2015-02-05 at 8 45 43 pm

### CODE REVIEW

**Comment by ngmiller** [comment]
@lighthouse/owners Putting this up for an initial review. There is still some loose ends I would like to tie up this weekend in terms of how often docker requests are made. But, if you have a chance to pull this down and try to break it, I would greatly appreciate it. Let me know if anything seems off or if you have UI suggestions!

File: app/js/instances/containerController.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    $scope.host = $routeParams.host;
+    $scope.id = $routeParams.id;
+
+    $scope.$listenTo(instanceModel, function () {
+        $scope.info = instanceModel.getContainer($scope.id);
+
+        if ($scope.info) {
+            var dState = $scope.info.State;
+            $scope.state = {
+                running: dState.Running && !dState.Paused,
```
Comment (ngmiller) [inline app/js/instances/containerController.js]: This is a slight workaround. Docker considers a paused container be both paused *and* running within its state model. It makes sense, but doesn't bode well with the needs of the UI.

**Comment by cmbrose** [comment]
Will dig a little more, but so far the only thing is this OCD nightmare
panic

**Comment by ngmiller** [comment]
@cmbrose Rest assured :) fixed the formatting

**Comment by zach-taylor** [comment]
Very nice

+1

**Comment by robertsheehy-wf** [comment]
+1

**Comment by cmbrose** [comment]
Sorry, forgot to add my +1

**Pull request merged**